.github/workflows/tests.yml

```diff
@@ -92,7 +92,7 @@ jobs:
 
       - name: Skip all remotes on Windows
         if: runner.os == 'Windows'
-        run: echo 'DISABLE_REMOTES_ARG=--disable-remotes=all' >> $env:GITHUB_ENV
+        run: echo 'DISABLE_REMOTES_ARG=--disable-remotes=azure,gs' >> $env:GITHUB_ENV
```

Comment:
I forgot that we can also run `s3` tests runs on Windows.

Reply:
Maybe that's what causes performance issues. `azure` and `gs` are skipped as there's no docker in Windows.

Reply:
Yup, s3 tests slows down Windows tests.